dotCMS/src/integration-test/java/com/dotcms/util/IntegrationTestInitService.java

```diff
@@ -48,10 +55,61 @@ public void init() throws Exception {
             // Init other dotCMS services.
             DotInitializationService.getInstance().initialize();
 
+            if (Config.getBooleanProperty("dotcms.integrationtest.run.upgradetask", true)) {
+                this.tryUpgradeTask();
```

Comment:
what is this change for? Did you run all the IT locally after this change?

Reply:
the idea is to avoid the validation locally

Reply:
I think this logic will run for every BeforeClass in the way it is implemented. If it's just for a test, this call should be outside of the init() method

Reply:
if you want to avoid the validation locally, the default value should be false

Reply:
it's ok inside the init because it happens only once per alltests